efficientdet/efficientdet_arch.py

```diff
@@ -393,6 +598,39 @@ def build_class_and_box_outputs(feats, config):
   return class_outputs, box_outputs
 
 
+class Build_class_and_box_outputs(tf.keras.layers.Layer):
```

Comment:
python class names conventionally use CamelCase and not underscores, unless you have a specific reason not to follow the convention.